core/tx_pool_test.go

```diff
@@ -261,6 +261,52 @@ func TestInvalidTransactions(t *testing.T) {
 	}
 }
 
+func TestBlacklistedTransactions(t *testing.T) {
```

Comment:
what's the format of the blacklist file?

Reply:
better have a test example file for it.

Reply:
It's a newline delimited file of addresses. I'll add comments in the setup function of main